sql_generators/glean_usage/common.py

```diff
@@ -40,6 +40,8 @@
 
 APPS_WITH_DISTRIBUTION_ID = ("fenix",)
 
+APPS_WITH_PROFILE_GROUP_ID = ("firefox_desktop",)
```

Comment:
This could also be configured in here: https://github.com/mozilla/bigquery-etl/blob/7260510cc3947114a98a347691c271f4dea092ec/bqetl_project.yaml#L271